resources/qml/dialogs/ImageOverlay.qml

```diff
@@ -103,10 +138,16 @@ Window {
             // and we don't yet distinguish mice and trackpads on Wayland either
             acceptedDevices: PointerDevice.Mouse | PointerDevice.TouchPad
             target: imgContainer
+
+            onWheel: handlerContainer.snapImageRotation()
         }
 
         DragHandler {
             target: imgContainer
+            xAxis.enabled: imgContainer.physicalWidth > Screen.width
+            yAxis.enabled: imgContainer.physicalHeight > Screen.height
```

Comment:
Shouldn't we be comparing against the window size here in case some compositor ignores our full screen request?

Reply:
(Same in the places above)